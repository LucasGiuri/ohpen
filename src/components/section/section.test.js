import Section from './section';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';

describe('Section', () => {
  const text = 'title';
  const dataTestPrefix = 'dataTestPrefix';

  describe('Snapshots', () => {
    it('default snapshot render', () => {
      const tree = renderer.create(<Section>asd</Section>).toJSON();
      expect(tree).toMatchSnapshot();
    });
  
    it('with custom dataTestPrefix snapshot render', () => {
      const tree = renderer.create(<Section dataTestPrefix={dataTestPrefix}>asd</Section>).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
