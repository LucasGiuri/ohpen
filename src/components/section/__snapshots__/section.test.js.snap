// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Section Snapshots default snapshot render 1`] = `
<div
  className="sc-bdfBQB hjzOBS"
  data-test="section-section"
>
  <div
    className="sc-gsTEea liuxQj"
  >
    <div
      className="sc-dlfnuX hyHGqJ"
    >
      <img
        className="sc-hKgJUU husJpk"
        src="logo.png"
      />
    </div>
    asd
  </div>
</div>
`;

exports[`Section Snapshots with custom dataTestPrefix snapshot render 1`] = `
<div
  className="sc-bdfBQB hjzOBS"
  data-test="dataTestPrefix-section"
>
  <div
    className="sc-gsTEea liuxQj"
  >
    <div
      className="sc-dlfnuX hyHGqJ"
    >
      <img
        className="sc-hKgJUU husJpk"
        src="logo.png"
      />
    </div>
    asd
  </div>
</div>
`;
